[
    {
        "user": "UUCD6P13J",
        "type": "message",
        "ts": "1596007139.024100",
        "client_msg_id": "aad306ed-11ef-4f03-ab47-78b4c51595ef",
        "text": "Can i ignore these occasional SocketErrors in flumine? `SocketError: [Connect: 2002]: Socket [Errno 104] Connection reset by peer\"`",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g6898272ab81",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/6898272ab810e419c2c9ca9af1e53da8.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0012-72.png",
            "first_name": "Lee",
            "real_name": "Lee U",
            "display_name": "Lee",
            "team": "T4G9NBD2M",
            "name": "lunswor",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "mfVr4",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "Can i ignore these occasional SocketErrors in flumine? "
                            },
                            {
                                "type": "text",
                                "text": "SocketError: [Connect: 2002]: Socket [Errno 104] Connection reset by peer\"",
                                "style": {
                                    "code": true
                                }
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U015QMDDSP7",
        "type": "message",
        "ts": "1596012478.024600",
        "client_msg_id": "ecdca4a8-b3c7-45df-8280-4f1e8f0f9c8a",
        "text": "I have been getting these errors intermittently as well and have been ignoring them",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g232999cc026",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/232999cc026888da5959f96d96d38903.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0015-72.png",
            "first_name": "",
            "real_name": "L",
            "display_name": "L",
            "team": "T4G9NBD2M",
            "name": "leonng",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "AN3ny",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "I have been getting these errors intermittently as well and have been ignoring them"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "U4H19D1D2",
        "type": "message",
        "ts": "1596012503.025100",
        "client_msg_id": "212F0ABA-EB4C-4C3B-AB32-0255341A51C6",
        "text": "Yeah it’s just network or betfair dropping it, it reconnects for you ",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "137c5a3ef323",
            "image_72": "https:\/\/avatars.slack-edge.com\/2024-03-15\/6819395173841_137c5a3ef323f1944a1a_72.png",
            "first_name": "liam",
            "real_name": "liam",
            "display_name": "liam",
            "team": "T4G9NBD2M",
            "name": "liam",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "8L6",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "Yeah it’s just network or betfair dropping it, it reconnects for you "
                            }
                        ]
                    }
                ]
            }
        ],
        "reactions": [
            {
                "name": "+1",
                "users": [
                    "UUCD6P13J"
                ],
                "count": 1
            }
        ]
    },
    {
        "user": "UFTBRB3F1",
        "type": "message",
        "ts": "1596012612.026000",
        "client_msg_id": "e98ac096-f3b9-451f-ae42-43e9151559b0",
        "text": "Has anyone ever seen this?\n\nWhen replacing I get this error (very occasionally):",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g771e110be94",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/a771e110be945b86bede9d5c1bb551e3.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0024-72.png",
            "first_name": "",
            "real_name": "Newbie99",
            "display_name": "",
            "team": "T4G9NBD2M",
            "name": "vjmehra",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "Tj0Yf",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "Has anyone ever seen this?\n\nWhen replacing I get this error (very occasionally):"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "UFTBRB3F1",
        "type": "message",
        "ts": "1596012618.026200",
        "client_msg_id": "c1554009-2d22-440a-b900-d18113ac4a3d",
        "text": "```\nReplace Error: RELATED_ACTION_FAILED\nException in thread Thread-1:\nTraceback (most recent call last):\n  File \"\/usr\/lib64\/python3.7\/threading.py\", line 926, in _bootstrap_inner\n    self.run()\n  File \"\/usr\/lib64\/python3.7\/threading.py\", line 870, in run\n    self._target(*self._args, **self._kwargs)\n  File \"racing.py\", line 568, in combined_queue\n    replaced_orders_report = rf.replace_orders(market_id, replace_list, market_info, trading, filters)\n  File \"\/home\/ec2-user\/trading\/env\/projects\/racing_functions.py\", line 1941, in replace_orders\n    'selection_id': place_instruction_report.instruction.selection_id,\nAttributeError: 'NoneType' object has no attribute 'selection_id'```",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g771e110be94",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/a771e110be945b86bede9d5c1bb551e3.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0024-72.png",
            "first_name": "",
            "real_name": "Newbie99",
            "display_name": "",
            "team": "T4G9NBD2M",
            "name": "vjmehra",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "Qa6",
                "elements": [
                    {
                        "type": "rich_text_preformatted",
                        "elements": [
                            {
                                "type": "text",
                                "text": "\nReplace Error: RELATED_ACTION_FAILED\nException in thread Thread-1:\nTraceback (most recent call last):\n  File \"\/usr\/lib64\/python3.7\/threading.py\", line 926, in _bootstrap_inner\n    self.run()\n  File \"\/usr\/lib64\/python3.7\/threading.py\", line 870, in run\n    self._target(*self._args, **self._kwargs)\n  File \"racing.py\", line 568, in combined_queue\n    replaced_orders_report = rf.replace_orders(market_id, replace_list, market_info, trading, filters)\n  File \"\/home\/ec2-user\/trading\/env\/projects\/racing_functions.py\", line 1941, in replace_orders\n    'selection_id': place_instruction_report.instruction.selection_id,\nAttributeError: 'NoneType' object has no attribute 'selection_id'"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "UFTBRB3F1",
        "type": "message",
        "ts": "1596012629.026700",
        "client_msg_id": "4e936fae-e057-4923-909e-a58a347e9524",
        "text": "The replace however actually seems to work",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g771e110be94",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/a771e110be945b86bede9d5c1bb551e3.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0024-72.png",
            "first_name": "",
            "real_name": "Newbie99",
            "display_name": "",
            "team": "T4G9NBD2M",
            "name": "vjmehra",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "dO1",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "The replace however actually seems to work"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "UFTBRB3F1",
        "type": "message",
        "ts": "1596012648.027300",
        "client_msg_id": "2de864f3-d5e8-4d58-9136-0f45a91f1b6a",
        "text": "The error just suggests the instruction somehow isn't populated for the report",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g771e110be94",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/a771e110be945b86bede9d5c1bb551e3.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0024-72.png",
            "first_name": "",
            "real_name": "Newbie99",
            "display_name": "",
            "team": "T4G9NBD2M",
            "name": "vjmehra",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "CmdU",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "The error just suggests the instruction somehow isn't populated for the report"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "UFTBRB3F1",
        "type": "message",
        "ts": "1596012655.027600",
        "client_msg_id": "2ea722a6-3f98-47fb-8c93-bb65c015470f",
        "text": "Which is odd, but not the end of the world",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g771e110be94",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/a771e110be945b86bede9d5c1bb551e3.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0024-72.png",
            "first_name": "",
            "real_name": "Newbie99",
            "display_name": "",
            "team": "T4G9NBD2M",
            "name": "vjmehra",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "gFw",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "Which is odd, but not the end of the world"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "user": "UFTBRB3F1",
        "type": "message",
        "ts": "1596012677.028000",
        "client_msg_id": "9171a9b1-ee61-433c-847a-c03b526ab438",
        "text": "I couldn't find a 'RELATED_ACTION_FAILED' error code in the docs",
        "team": "T4G9NBD2M",
        "user_team": "T4G9NBD2M",
        "source_team": "T4G9NBD2M",
        "user_profile": {
            "avatar_hash": "g771e110be94",
            "image_72": "https:\/\/secure.gravatar.com\/avatar\/a771e110be945b86bede9d5c1bb551e3.jpg?s=72&d=https%3A%2F%2Fa.slack-edge.com%2Fdf10d%2Fimg%2Favatars%2Fava_0024-72.png",
            "first_name": "",
            "real_name": "Newbie99",
            "display_name": "",
            "team": "T4G9NBD2M",
            "name": "vjmehra",
            "is_restricted": false,
            "is_ultra_restricted": false
        },
        "blocks": [
            {
                "type": "rich_text",
                "block_id": "Sqt=b",
                "elements": [
                    {
                        "type": "rich_text_section",
                        "elements": [
                            {
                                "type": "text",
                                "text": "I couldn't find a 'RELATED_ACTION_FAILED' error code in the docs"
                            }
                        ]
                    }
                ]
            }
        ]
    },
    {
        "subtype": "channel_join",
        "user": "U0183C24N84",
        "text": "<@U0183C24N84> has joined the channel",
        "type": "message",
        "ts": "1596013940.028200"
    }
]